playwright/tests/screenshots/scripting.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test, type Page } from '@playwright/test';
+import type { Accordion } from '../pageobjects/Accordion';
+import { InscriptionView } from '../pageobjects/InscriptionView';
+import { screenshot } from './screenshot-util';
+
+const GENERIC_PID = {
+  SCRIPT: '168F0C6DF682858E-f3',
+  USER_TASK: '168F0C6DF682858E-f5'
+} as const;
+
+test.describe('Scripting', () => {
+  test('Completion', async ({ page }) => {
+    await page.setViewportSize({ width: 500, height: 600 });
+    const accordion = await openAccordion(page, GENERIC_PID.SCRIPT, 'Output');
+    const section = await codeOnly(accordion);
+    const script = section.scriptArea();
+
+    await script.fill('ivy.');
+    await script.triggerContentAssist();
+    await screenshot(section.currentLocator(), 'code-completor.png');
+  });
+
+  test('Hopping', async ({ page }) => {
+    await page.setViewportSize({ width: 500, height: 600 });
+    const accordion = await openAccordion(page, GENERIC_PID.SCRIPT, 'Output');
+    const section = await codeOnly(accordion);
+    const script = section.scriptArea();
+
+    await script.fill('in.set');
+    await script.triggerContentAssist();
+    await page.keyboard.press('Enter');
+
+    await screenshot(section.currentLocator(), 'code-param-hopping.png');
+  });
+
+  test('Macro', async ({ page }) => {
+    await page.setViewportSize({ width: 500, height: 600 });
+    const accordion = await openAccordion(page, GENERIC_PID.USER_TASK, 'Task');
+    const section = accordion.section('Details');
+    await section.open();
+    const script = section.macroArea('Name');
+
+    await script.fill('Verify User <%= ');
+    await script.focus();
+    await page.keyboard.press('ArrowLeft+ArrowLeft+ArrowLeft');
+    await page.keyboard.type('in.');
```

Comment:
looks like belonging into a page-object; but feels like a hack hard to name and make reproducible.
I'm just doing this stuff to navigate into a macro expression; showing its content completion